services/repository/main_test.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package repository
+
+import (
+	"path/filepath"
+	"testing"
+
+	"code.gitea.io/gitea/models"
+)
+
+func TestMain(m *testing.M) {
+	models.MainTest(m, filepath.Join("..", ".."))
```

Comment:
why this test is needed?

Reply:
Because https://github.com/go-gitea/gitea/pull/8573/files#diff-e033c84bd172bdc2d4381884cb7fe41cR27 need this.